src/css/profile/mobile/base.less

```diff
@@ -76,6 +76,8 @@
 @slider-value-color: @slider-base-color;
 @slider-handler-color: @slider-base-color;
 @content-area-line-color: #d6d6d6;
+
+@dropdown-list-margin: 6 * @px_base;
 @dropdown-list-box-shadow-color: fade(@_black, 35%);
```

Comment:
Try to find and remove this style:
![image](https://user-images.githubusercontent.com/29534410/77542166-0bcf8f80-6ea6-11ea-8ea9-6f37752442ed.png)

Reply:
Done.